### ISSUE TITLE
No mobi version in 05 & PDF too large for Kindle

### ISSUE THREAD

**Issue by genehack** [robconery/feedback#174]
Hi - The 05 ZIP I downloaded doesn't have a .mobi version (only .epub and .pdf). Further, the PDF is ~53MB, which means the Send To Kindle app won't load it (it has a 50MB limit). 

Hopefully the final will have the .mobi version back?

**Comment by robconery** [comment]
Hi there - did you read the emails sent prior to the drop and with the drop itself? These issues are explained there. The short end is that I can't get the mobi version below 700Mb for some reason and, yes, the PDF is rather large. Nothing I can do about that.

**Comment by robconery** [comment]
Also - this issue list is for issues with the content of the book; support stuff can go directly to my email.

**Comment by genehack** [comment]
Yeah, I did read the email (to be exact, the email with the subject line "Imposter's Handbook Drop #5 Will Be Ready In The Next 24 Hours"). In fact, I filed the issue here because the email specifically said:

> As always, if you have any suggestions or see any problems – be sure to let me know by filing an issue here. It works sooo much better to have them logged on Github rather than Twitter or Email.

(Where "filing an issue here" being a link to the Github issues page for this repo.)

You also didn't say _anything_ in that email about the .mobi version, other than mentioning using a new tool to do the layout: 

> With respect to the mobi files (for Kindle/Nook/etc): I used Amazon's new Kindle emulator to view and then reformat the epub file I generated. It looked _perfect_ in the emulator, but like utter crap on a Kindle (go figure). I gave it another go with a different editor (Scrivener) and I think it looks rather nice,  but if you see problems with the mobi file... I would suggest reading the PDF.

Finally, not to be bitchy, but "if you see problems with the mobi file" _strongly_ suggests to me that I should expect to see a .mobi file in the ZIP. When I didn't, I assumed there might have been a problem with the Zip file preparation, and thought I'd be helpful by asking about it -- since nobody else had yet. I guess maybe I understand why now.

**Comment by robconery** [comment]
I should have been more clear: _the email accompanying the drop_, wherein I wrote:

> Unfortunately, for some very, very strange reason, Kindlegen (the app that makes the mobi files out of the epub) is refusing to play nice and, for some reason, is generating a 750Mb file! So, I'm hoping that the PDF version will suffice - there is no mobi file and, to be honest, I don't think there ever will be again. It's just too much work to figure out a proprietary format.

When I wrote the notification (prior to sending out), I was _assuming_ (which I shouldn't have) that the mobi format would work. It didn't.

I should also be a bit clearer RE "filing an issue" - I'm trying to use this issue list for content, not support in this sense. If you don't mind, I'd like to take this to email (just respond to any of the notifications) and we can sort it there.

To directly answer you: _I'm trying to get f_*_ing mobi format in place_. It's causing me massive headaches and this whole ebook formatting thing is taking years off my life :).
